docs/source/user/application-structure.rst

```diff
@@ -3,3 +3,143 @@ Application Structure
 
 .. contents:: |toctitle|
     :local:
+
+.. _accuracy:
+
+Accuracy
+--------
+
+The library contains utilities to estimate accuracy at a given noise scale and statistical significance level,
+or derive the necessary noise scale to meet a given target accuracy and statistical significance level.
+
+The noise scale may be either laplace or gaussian.
+
+:laplacian: | Applies to any L1 noise addition mechanism.
+  | :func:`make_base_laplace() <opendp.meas.make_base_laplace>`
+  | :func:`make_base_geometric() <opendp.meas.make_base_geometric>`
+  | :func:`make_base_stability(MI=L1Distance[T]) <opendp.meas.make_base_stability>`
+:gaussian: | Applies to any L2 noise addition mechanism.
+  | :func:`make_base_gaussian() <opendp.meas.make_base_gaussian>`
+  | :func:`make_base_stability(MI=L2Distance[T]) <opendp.meas.make_base_stability>`
+
+The library provides the following functions for converting to and from noise scales:
+
+* :func:`opendp.accuracy.laplacian_scale_to_accuracy`
+* :func:`opendp.accuracy.accuracy_to_laplacian_scale`
+* :func:`opendp.accuracy.gaussian_scale_to_accuracy`
+* :func:`opendp.accuracy.accuracy_to_gaussian_scale`
+
+These functions take either scale or accuracy, and alpha, a statistical significance parameter.
+
+You can generally plug the distribution, scale, accuracy and alpha
+into the following statement to interpret these functions:
+
+.. code-block:: python
+
+    f"When the {distribution} scale is {scale}, "
+    f"the DP estimate differs from the true value by no more than {accuracy} "
+    f"at a statistical significance level alpha of {alpha}, "
+    f"or with (1 - {alpha})100% = {(1 - alpha) * 100}% confidence."
+
+.. _binary-search:
+
+Binary Search
```

Comment:
I'd call this Choosing Parameters or Choosing Noise Scale.

Also, I think this section should come first, as it's more fundamental than accuracy.

Reply:
I don't like "choosing noise scale" because it's really "choosing any free parameter". How about "Parameter Search"?